src/test/groovy/bugs/Groovy7909Bug.groovy

```diff
@@ -0,0 +1,57 @@
+package groovy.bugs
```

Comment:
Could you add the license header to this file, should be able to grab it from another file in the same package.